src/main/java/com/mitchej123/hodgepodge/Compat.java

```diff
@@ -30,11 +30,22 @@ static void init(Side side) {
         }
     }
 
+    /**
+     * Cannot be used before pre-init phase.
+     */
     public static boolean isNeiLeftPanelVisible() {
         return isNeiPresent
                 && isClient
                 && NEIClientConfig.isEnabled()
                 && !NEIClientConfig.isHidden()
                 && (!doesNeiHaveBookmarkAPI || !NEIClientConfig.isBookmarkPanelHidden());
     }
+
+    /**
+     * Cannot be used before mod construction phase.
+     */
+    public static boolean isGT5Present() {
+        ModContainer gtModContainer = Loader.instance().getIndexedModList().get("gregtech");
+        return gtModContainer != null && gtModContainer.getVersion().equals("MC1710");
```

Comment:
Another option would be to check `Loader.isModLoaded("gregtech") && !Loader.isModLoaded("gregapi_post")`, since only GT6 provides `gregapi_post`. Maybe a bit cleaner.

Reply:
You could completely remove this method if you add GT6 to `TargetedMod` (with `gregapi_post` as modid) and set it as excluded mod in the `Mixins` enum.

Reply:
That does sound simpler. Any particular reason for choosing `gregapi_post` instead of `gregapi` though? The latter would be more straightforward.

Reply:
I don't know if GT6 has `gregapi` aswell, I went off of https://github.com/GTNewHorizons/NotEnoughItems/blob/master/src/main/resources/assets/nei/csv/handlers.csv#L91 (see the "excludedModId" column).

Reply:
GT6 has three modids: `gregtech`, `gregapi`, `gregapi_post`. I'll go with `gregapi` then since it seems to work fine.